worlds/sc2/mission_order/types.py

```diff
@@ -4,10 +4,6 @@
 
 from .structs import LayoutType, SC2MOGenMission
 
-def _empty_slot() -> SC2MOGenMission:
-    slot = SC2MOGenMission(None, None, set())
-    slot.option_empty = True
-    return slot
```

Comment:
Leave this please, I've left this in preparation for layout types that actually have empty spaces by default. I'm not 100% sure it'll be needed, but I wanted to have something usable ready.

Reply:
kk, but be aware that mypy lights it up with some of the highest type error density in the sc2/ folder.